java/ql/src/experimental/CWE-939/IncorrectURLVerification.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @id java/incorrect-url-verification
+ * @name Insertion of sensitive information into log files
+ * @description Apps that rely on URL parsing to verify that a given URL is pointing to a trusted server are susceptible to wrong ways of URL parsing and verification. 
+ * @kind problem
+ * @tags security
+ *       external/cwe-939
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+import DataFlow
+import PathGraph
+
+
+/**
+ * The Java class `android.net.Uri` and `java.net.URL`.
+ */
+class Uri extends RefType {
+    Uri() {
+        hasQualifiedName("android.net", "Uri") or
+        hasQualifiedName("java.net", "URL")
+    }
+}
+
+/**
+ * The method `getHost()` declared in `android.net.Uri` and `java.net.URL`.
+ */
+class UriGetHostMethod extends Method {
+    UriGetHostMethod() {
+        getDeclaringType() instanceof Uri and
+        hasName("getHost") and
+        getNumberOfParameters() = 0
+    }
+}
+
+/**
+ * A library method that acts like `String.format` by formatting a number of
+ * its arguments according to a format string.
```

Comment:
Old comment here.

Reply:
Removed as well.

Regards,
@luchua-bc

Reply:
It looks like this is still the case.

Reply:
I've corrected the comment and committed the change. Sorry about this.

@luchua-bc